code/modules/mechs/components/body.dm

```diff
@@ -254,9 +237,11 @@
 	icon_state = "light_body"
 	max_damage = 75
 	power_use = 5
+	emp_shielded = TRUE
 	climb_time = 10 //gets a buff to climb_time, in exchange for being less beefy
 	has_hardpoints = list(HARDPOINT_BACK, HARDPOINT_RIGHT_SHOULDER)
-	matter = list(MATERIAL_STEEL = 20, MATERIAL_GLASS = 5, MATERIAL_PLASTIC = 10)
+	matter = list(MATERIAL_STEEL = 20, MATERIAL_GLASS = 6, MATERIAL_PLASTIC = 10)
+	armor = list(melee = 16, bullet = 8, energy = 4, bomb = 40, bio = 100, rad = 100)
 
```

Comment:
No mention in the component description about light parts not being affected by direction.

Reply:
Examine data is added now